plugins/home/backstage-plugin-home-markdown/src/MarkdownCard/types.ts

```diff
@@ -24,7 +24,7 @@ export type MarkdownContentProps = {
   repo: string;
   path: string;
   branch?: string;
-  purgeHtmlComments?: boolean;
+  dontStripHtmlCommentsBeforeRendering?: boolean;
```

Comment:
can this break customer's tenants?

Reply:
No, not really. its very specific and most people will want these comments gone.